src/i18n/locale-de.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I would prefer the labels to be in capital letters for better detection in the code. They are also constants.

Reply:
And I would prefer them to be hierarchically structured. But as this is just starting out, it can be changed down the road.

Reply:
I agree. We already discussed this and I'll change them with the next MR